src/MagnumPlugins/KtxImageConverter/Test/KtxImageConverterTest.cpp

```diff
@@ -197,7 +217,7 @@ const struct {
     {"ETC2", "1d-compressed-etc2.ktx2", CompressedPixelFormat::Etc2RGB8Srgb, {7}}
 };
 
-const struct {
+constexpr struct {
```

Comment:
FYI I don't really bother with marking static data as `constexpr` in tests anymore, because I have a hunch that it might have a negative effect on compile times, due to the compiler having to execute all that C++ code through a slow interpreter. (And it causes nasty issues or crashes in older compilers.) For tests I don't care much about execution startup time, iteration time is a bigger priority.

Didn't really have time to measure the impact yet, but for new code -- unless the `constexpr` is needed to actually test a `constexpr` feature -- I just slap on a `const` and that's it. If there's already a `constexpr` and it doesn't cause issues I leave it as-is.

Reply:
Noted for next time. The Ktx{Importer, ImageConverter} and CgltfImporter tests are all plastered with `constexpr`, I figure a concerted update for all of those in the future makes more sense than reverting the change just here.